src/main/drivers/pwm_output.c

```diff
@@ -386,6 +386,11 @@ uint32_t getDshotHz(motorPwmProtocolTypes_e pwmProtocolType)
     }
 }
 
+FAST_RAM bool pwmIsProcessingDshotCommand(void)
```

Comment:
`FAST_RAM`? I'm not sure it'll even work in the end, DTCM-RAM and CCM are not executable.
Should be `FAST_CODE` if you're sure